build/patch-charts.sh

```diff
@@ -513,6 +513,39 @@ function copyGlobalValues() {
   cp ${SOURCE_DIR}/resources/helm/overlays/global.yaml ${SOURCE_DIR}/resources/helm/v2.1/
 }
 
+# This hack is hopefully only needed for a few versions until this PR is merged: https://github.com/istio/istio/pull/37264
```

Comment:
I would prefer for these changes to be made in the istio charts directly, and then cherry-picked to maistra/istio

Reply:
It has been a struggle to get any progress on this PR, I am now getting core security contributors reviewing it, as soon as I get that merged I will come back and report here.

Reply:
https://github.com/istio/istio/pull/38845

Reply:
it looks like I will have to change extraArgs a little but that should be a straightforward change..